app/templates/buy-quick-start.pug

```diff
@@ -1,12 +1,12 @@
 .quick-start.col-md-6.col-sm-12.col-xs-12.pln.prn
   .pt-30.pb-30.border-desktop(ng-class="{'ph-15': !inMobileBuy}")
-    form(name="fiatForm" ng-submit="!status.busy && fiatForm.$valid && triggerBuy()" novalidate)
+    form(name="fiatForm" ng-submit="!status.busy && fiatForm.$valid && buy({quote: quote})" novalidate)
       .flex-row.flex-between
         .flex-center
           label.em-500(translate="ENTER_AMOUNT:")
-          helper-button.mbs(ng-disabled="false" ng-show="limits.card.min && limits.card.max" content="MAX_MIN_HELPER" values="{cardMax: limits.card.max, min: limits.card.min, bankMax: limits.bank.max, symbol: currencySymbol.symbol}")
+          helper-button.mbs(ng-disabled="false" ng-show="exchangeRate.fiat" content="MAX_MIN_HELPER" values="{max: limits.absoluteMax(transaction.currency.code), min: limits.absoluteMin(transaction.currency.code), code: transaction.currency.code}")
```

Comment:
Maybe move that complex `max:` `min:` object to the controller?
`content="MAX_MIN_HELPER" values = "$ctrl. ..."`?